articles/install-guide/qjupyter.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Develop with Q# Jupyter notebooks
+author: natke
+ms.author: nakersha
+ms.date: 9/30/2019
+ms.topic: article
+ms.custom: how-to
+uid: microsoft.quantum.install.jupyter
+---
+
+# Develop with Q# Jupyter notebooks
+
+> [!WARNING]
+> * In Q# Jupyter Notebooks you can only run Q# code, and the operations cannot be called from external host programs (e.g. Python or C# files). This environment is not appropriate if your goal is to combine an external classical host program with the quantum program.
+
+Commonplace in academic settings, scientific labs, and online-based collaborative programming, Jupyter Notebooks allow in-place code execution alongside instructions, notes, and other content. This environment is great for writing Q# code with embedded explanations or quantum computing interactive tutorials. Here's what you need to do to start creating your own Q# notebooks.
```

Comment:
This para sounds way too sales-y. Suggest toning it down.

Reply:
Addressed in 74a7a54